app/controllers/experiments_controller.rb

```diff
@@ -412,14 +412,19 @@ def create
           inf[:infrastructure][:params][:credentials_id] = BSON::ObjectId(inf[:infrastructure][:params][:credentials_id])
         end
       end
+      planned_finish_time = Time.now + workers_scaling_params[:time_limit]*60 # minutes to seconds
```

Comment:
.minutes zamiast *60, ale poprawiłem to u siebie przy okazji. Przy mergowaniu trzeba pamiętać żeby nie przepadło.